cypress/lib/test-fixture-handler.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export class TestFixtureHandler {
+  constructor(inputTestRunner, openSearchUrl = 'localhost:9200') {
```

Comment:
we aren't we pulling from the cypyress environment variable ?

Reply:
updated